vnext/PropertySheets/External/Microsoft.ReactNative.Uwp.CppLib.targets

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- 
+  Copyright (c) Microsoft Corporation. All rights reserved.
+ Licensed under the MIT License.. 
```

Comment:
nit: please fix the formatting in in all headers: indentation and remove extra dot and space in the end.

Reply:
copy-paste error. Updated all.

---
In reply to: [428723258](https://github.com/microsoft/react-native-windows/pull/4961#discussion_r428723258) [](ancestors = 428723258)